### DISCUSSION TITLE
Updating Cron Schedules

### DISCUSSION THREAD

**Discussion by TheAccidentalDBA** [trimble-oss/dba-dash#172] [Q&A] [answered]
Hello!

I'm trying to update the cron schedule to every five minutes.  This is the syntax I'm using: */5 * * * *

The scheduler bounces an "invalid cron expression" error.  Is this syntax incorrect?

![image](https://user-images.githubusercontent.com/71884153/165520047-0783ac89-6f9b-4bc2-a720-1bd63121e32f.png)

Thank you!

**Comment by TheAccidentalDBA** [comment]
I was able to figure this issue out.  Here is the syntax used in case anyone else is interested:

 0 0,5,10,15,20,25,30,35,40,45,50,55 * ? * * *

**Comment by DavidWiseman** [comment, marked as answer]
You can also use `0 0/5 * ? * * *` for every 5min.  Or you can use `300`  (value in seconds instead of a cron expression).

Note: A value in seconds (300) will execute at a random time depending on when the service starts, repeating at the specified interval.  The cron expression will repeat every 5min on the 5min mark.

I've just noticed the linked cronmaker website has disappeared.  I'll get that updated.  It is a bit of a cryptic format so having a builder helps!  I found [this alternative](https://www.freeformatter.com/cron-expression-generator-quartz.html) after a quick google.

**Comment by DavidWiseman** [comment]
I've [updated the docs](https://github.com/trimble-oss/dba-dash/blob/main/Docs/Collection.md#cron-expressions) and the broken link to cronmaker will be removed in the next release.
